src/components/GettingStartedDropdown/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  Button,
+  Menu,
+  MenuButton,
+  MenuDivider,
+  MenuGroup,
+  MenuItem,
+  MenuList,
+} from "@chakra-ui/react";
+import { GETTING_STARTED } from "../../constants/links";
+import { ExternalLink } from "../ExternalLink";
+
+export default function GettingStartedDropdown(): JSX.Element {
+  return (
+    <Menu colorScheme="#002954">
```

Comment:
Couple of things:
- The pattern we're using for files is to name components as `Component.tsx` & use an `index.ts`
- We've been preferring named exports rather than default
- We should be using colors from theme as much as possible rather than a hex value

Reply:
How do i find the themes we use?

Reply:
nvm i found it!